src/constants/index.ts

```diff
@@ -712,6 +738,7 @@ export const ARB_USD_POOL_TOKENS = [NUSD, MIM, USDC, USDT]
 export const OPT_USD_POOL_TOKENS = [DAI, USDC, USDT]
 export const FRAX_OPT_USD_POOL_TOKENS = [FRAX, ...OPT_USD_POOL_TOKENS]
 export const FRAX_OPT_USD_UNDERLYING_POOL_TOKENS = [FRAX, OPT_USD_SWAP_TOKEN]
+export const FTM_USD_POOL_TOKENS = [FRAX, USDC, DAI]
```

Comment:
Order is now FRAX, DAI, USDC https://github.com/saddle-finance/saddle-contract/pull/510/files#diff-746e17d3e6cf7292c9c1761afdaec7fb7773feb55a3336101c288f95ab4c0904R6